<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_connexion">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Identifiez vous:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" colspan="4">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QLineEdit" name="pseudo_l"/>
           </item>
           <item row="2" column="0" colspan="4">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Mot de passe:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="3">
            <widget class="QLineEdit" name="pass_l">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="4">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QPushButton" name="inscription_b">
             <property name="styleSheet">
              <string notr="true">color:blue;</string>
             </property>
             <property name="text">
              <string>Pas encore inscrit ? Faites-le maintenant en moins de 2 minutes !</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" rowspan="2" colspan="4">
            <widget class="QPushButton" name="valider_b">
             <property name="text">
              <string>Connexion !</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="7" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Pseudo:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_inscription">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Inscription:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QWebView" name="vueInternet">
             <property name="url">
              <url>
               <string>http://localhost/ecopy/inscription.php</string>
              </url>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="retour_b">
             <property name="text">
              <string>Retour</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_main">
       <layout class="QGridLayout" name="gridLayout_6" rowstretch="1,5" columnstretch="2,1">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="info_l">
          <property name="text">
           <string>Que faire ?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Chercher quelqu'un ...</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7" columnstretch="1,0,1">
           <item row="0" column="0">
            <widget class="QLineEdit" name="pseudo_search_l">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="chercher_b">
             <property name="text">
              <string>Chercher</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="4">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="fichier_b">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Envoyer un fichier</string>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="message_b">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Discuter</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" rowspan="2" colspan="2">
            <widget class="QGroupBox" name="infos">
             <property name="title">
              <string>Informations</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Pseudo :</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2" colspan="4">
               <widget class="QLineEdit" name="found_pseudo">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>IP :</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Port :</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Connecté :</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2" colspan="4">
               <widget class="QLineEdit" name="found_port">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="2" colspan="4">
               <widget class="QLineEdit" name="found_co">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="2" colspan="4">
               <widget class="QLineEdit" name="found_ip">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>eCopy - Client BETA 0.1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuECopy">
    <property name="title">
     <string>eCopy</string>
    </property>
    <addaction name="actionReduire"/>
    <addaction name="actionSe_reconnecter"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuECopy"/>
  </widget>
  <widget class="QStatusBar" name="status"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionSe_reconnecter">
   <property name="text">
    <string>Se reconnecter</string>
   </property>
  </action>
  <action name="actionReduire">
   <property name="text">
    <string>Réduire</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pseudo_l</tabstop>
  <tabstop>pass_l</tabstop>
  <tabstop>valider_b</tabstop>
  <tabstop>inscription_b</tabstop>
  <tabstop>pseudo_search_l</tabstop>
  <tabstop>chercher_b</tabstop>
  <tabstop>fichier_b</tabstop>
  <tabstop>message_b</tabstop>
  <tabstop>vueInternet</tabstop>
  <tabstop>retour_b</tabstop>
  <tabstop>found_pseudo</tabstop>
  <tabstop>found_port</tabstop>
  <tabstop>found_co</tabstop>
  <tabstop>found_ip</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pseudo_search_l</sender>
   <signal>returnPressed()</signal>
   <receiver>chercher_b</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
